rest-client/src/main/java/org/eclipse/dataspaceconnector/registration/client/ApiClientFactory.java

```diff
@@ -14,23 +14,34 @@
 
 package org.eclipse.dataspaceconnector.registration.client;
 
+import org.eclipse.dataspaceconnector.spi.EdcSetting;
 import org.eclipse.dataspaceconnector.spi.iam.TokenParameters;
 import org.eclipse.dataspaceconnector.spi.iam.TokenRepresentation;
 import org.eclipse.dataspaceconnector.spi.result.Result;
 import org.jetbrains.annotations.NotNull;
 
+import java.time.Duration;
 import java.util.function.Function;
 
+import static org.eclipse.dataspaceconnector.common.configuration.ConfigurationFunctions.propOrEnv;
+
 /**
  * Factory class for {@link ApiClient}.
  */
 public class ApiClientFactory {
+
+    @EdcSetting
```

Comment:
sorry to be nit-picky, but the @EdcSetting annotation now has a few attributes, please do use them.

Reply:
Updated. Thanks.